### PR TITLE
APIGOV-24234 Edit assets and products without requiring a new re…

### PR DESCRIPTION
**Pull request by lbadenhop**
…lease

Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes here to communicate to the maintainers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

File: content/en/docs/manage_asset_catalog/asset_management.md [hunk 1]
```diff
@@ -89,10 +90,11 @@ To create an asset:
...
-9. Provide the Category, Tags, and Attribute details for the asset. Click **Save**.
-10. Click **Save Draft and Exit** to exit Asset Creation.
+9. Provide the Category, Tags, and Attribute details for the asset. Click **Next**.
+10. At the **Preview Asset** step, you can view the Asset details prior to creation.  
+At his point you can click on **Save** to create the asset in a draft state.  Alternatively, you can click on **Save & Activate** to create a new asset version release in an **Active** state for consumption in a product definition.
 
-The asset is created in **Draft** state. To use this asset in a product definition, you must move the asset to an **Active** state by activating the asset.
+If the asset is created in **Draft** state. To use this asset in a product definition, you must move the asset to an **Active** state by activating the asset.
 
 ## Activate an asset
```
Comment (georgiana-caras) [inline content/en/docs/manage_asset_catalog/asset_management.md:99]: @robertengaxway , maybe we should rename this section to Edit an asset to better reflect the content of the section. The content describes more how to edit the asset.

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

**Comment by lbadenhop** [comment]
@robertengaxway Please review my edits.

File: content/en/docs/manage_asset_catalog/asset_management.md (hunk 1 above)
Comment (lbadenhop) [reply content/en/docs/manage_asset_catalog/asset_management.md:98]: I don't understand why you want to rename Activate an Asset to Edit and asset. There is already an Edit an Asset section.

Comment (robertengaxway) [reply content/en/docs/manage_asset_catalog/asset_management.md:98]: @georgiana-caras  The "Edit an asset" is subsection of "Asset Management".   There are other subsections for create, activate, and organize.   Activation is a separate action after creating a draft or changing an asset from Deprecated.   
I think the current "Edit an asset" subsection title is in the correct place

**Review by robertengaxway — APPROVED** [review]
I approve

File: content/en/docs/manage_asset_catalog/asset_management.md [hunk 2]
```diff
@@ -83,16 +85,16 @@ To create an asset:
...
+    * Auto-Release - Enabled/Disabled of the auto release of new Asset versions.  If auto-release is **enabled**, a new asset will be automatically released with each change to any one of the grouped resources (i.e. API services).   For example, auto-release enabled will trigger an automatic asset release every time a new grouped API Service revision is discovered or created. This would enable a more automated and hands-off experience.   If you want more control of the asset release process, select auto-relase to **disabled**.
+    * Auto-Release Type -  The default is set to a Patch release versions and you can select another type of asset release versioning (i.e. major or minor).
 
 6. In Group Resources, find the available API resources you want to link, and click on the Link icon. For each resource, select the appropriate version. Default is **Latest**. Click **Link Versions**.
 7. Configure the Access Request handling, Manual or Automatic, and click **Next**.
```
Comment (georgiana-caras) [inline content/en/docs/manage_asset_catalog/asset_management.md:92]: @robertengaxway , you could explain what the Access Request handling is for, the way you did for the auto-release.

File: content/en/docs/manage_asset_catalog/asset_management.md [hunk 3]
```diff
@@ -83,16 +85,16 @@ To create an asset:
...
-8. Add the Access Rights for the asset, and click **Next**.
-9. Provide the Category, Tags, and Attribute details for the asset. Click **Save**.
-10. Click **Save Draft and Exit** to exit Asset Creation.
+8. At the Access Rights step, select the team(s) the asset can be shared with. By default, an asset is not shared and only the **Central Admin** will have access to it. If you shared your asset with a specific team, each member of the team will be able to see your product. Click **Next**.
```
Comment (georgiana-caras) [inline content/en/docs/manage_asset_catalog/asset_management.md:93]: team will be able to see your asset (not product)

File: content/en/docs/manage_product_foundry/foundry_product_management.md [hunk 4]
```diff
@@ -8,16 +8,17 @@ Manage your products, including: create, activate a release, edit, delete, monet
 
 ## Before you start
 
-You must have credentials or a user account from your org administrator to use the Product Catalog WebUI. Please follow the steps [here](https://docs.axway.com/bundle/platform-management/page/docs/management_guide/organizations/managing_organizations/index.html#managing-users).
+You must have credentials or a user account from your org administrator to use the Product Foundry WebUI. Please follow the steps [here](https://docs.axway.com/bundle/platform-management/page/docs/management_guide/organizations/managing_organizations/index.html#managing-users).
```
Comment (georgiana-caras) [inline content/en/docs/manage_product_foundry/foundry_product_management.md:11]: To be replaced with: You must have the Catalog Manager team role assigned by tour organization administrator in order to use the Product Foundry. Please follow the steps [here](https://docs.axway.com/bundle/platform-management/page/docs/management_guide/organizations/managing_organizations/index.html#managing-users).

File: content/en/docs/manage_product_foundry/foundry_product_management.md [hunk 5]
```diff
@@ -87,22 +89,29 @@ View the following detailed information for a specific product in any state (Dra
...
-6. Select a Usage Plan. One free plan will be automatically added to the product. Click **Next**.
-7. Add the quota to the plan. By default, one quota is added. You can assign multiple resource to the quota, but each resource can be part of only one quota. To add a quota, click **+ Add Quota** and select the resources and the limit for the quota. Click **Next**.
-8. Add documentation elements (sections and articles). By default, an empty document is created.
+5. Add the following Release Management information and then click **Next**:
+
+    * Auto-Release - Enabled/Disabled of the auto release of new Product versions. If auto-release is **enabled**, a new product will be automatically released with each change to any one of the linked resources (i.e. Assets). For example, auto-release enabled will trigger an automatic prodcut release every time a new grouped asset version is discovered or created. This would enable a more automated and hands-off experience. If you want more control of the product release process, select auto-relase to **disabled**.
```
Comment (georgiana-caras) [inline content/en/docs/manage_product_foundry/foundry_product_management.md:94]: Typo prodcut

**Review by georgiana-caras — CHANGES_REQUESTED** [review]
Overall there are not many things to change, see inline comments. For the product foundry most of the text in preview appears bold, there seems to be an syntax error in the markdown file

File: content/en/docs/manage_asset_catalog/asset_management.md (hunk 2 above)
Comment (robertengaxway) [reply content/en/docs/manage_asset_catalog/asset_management.md:92]: done

File: content/en/docs/manage_asset_catalog/asset_management.md (hunk 3 above)
Comment (robertengaxway) [reply content/en/docs/manage_asset_catalog/asset_management.md:93]: fixed

File: content/en/docs/manage_product_foundry/foundry_product_management.md (hunk 4 above)
Comment (robertengaxway) [reply content/en/docs/manage_product_foundry/foundry_product_management.md:11]: done

File: content/en/docs/manage_product_foundry/foundry_product_management.md (hunk 5 above)
Comment (robertengaxway) [reply content/en/docs/manage_product_foundry/foundry_product_management.md:94]: done

**Review by robertengaxway — APPROVED** [review]

**Pull request merged**